src/video/vita/SDL_vitagl_pvr.c

```diff
@@ -77,9 +77,8 @@ int VITA_GL_LoadLibrary(_THIS, const char *path)
     SDL_snprintf(hint.szWindowSystem, MAX_PATH, "%s/%s", default_path, "libpvrPSP2_WSEGL.suprx");
 
     PVRSRVCreateVirtualAppHint(&hint);
-}
```

Comment:
This removal is wrong. The issue is at line 27: brace should be moved before the line comment.

Reply:
You meant line 56?

Reply:
The previous builds failed because we had `if (xxx) // comment {`
The correct form should be `if (xxx) { //comment`

> You probably meant line 56?

Oh yes. (how did I invent 27...)

Reply:
For some, 42 is an important number. For others it's 69. For you, it's 27 :P

Reply:
I have force pushed new commits.